extension/src/path/selection/model.ts

```diff
@@ -49,33 +49,35 @@ export abstract class PathSelectionModel<
       .map(element => ({ ...element, selected: !!this.status[element.path] }))
   }
 
-  public toggleStatus(path: string) {
+  public toggleStatus(path: string, ...args: unknown[]) {
```

Comment:
Wrap `getTerminalNodes`, `getTerminalNodeStatuses`, etc in `PathsModel`  so that we don’t have to mess with the base class

Reply:
Can't maintain this class without type safety.